### ISSUE TITLE
'(1)' names cannot join bots - Player Name Validation Fails

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#5287]
**Describe the bug**
Joining a bot fails when player name is suffixed with " (1)"

**To Reproduce**
- start lobby and headless bot locally
- start two headed game clients
- log in with the same name to lobby (notice one name is suffixed with " (1)"
- join the bot game with the " (1)" player
- notice: error message that player name is not allowed

**Expected behavior**
- Should be able to join the game, no error message

**Additional context**

Moving player name management to http server could fix this.  Http server will likely not need the " (1)" suffix. Namely you could log in multiple times, and if so you would keep the same name across the multiple logins. The multiple login would be transparent to other users, it would appear as if you are just one player even though you would have multiple chat windows open. This can be done with http server as the player name is not needed for message routing, with the java server message routing is based on names and they need to be unique.

**Comment by prastle** [comment]
Please explain further.  Or in layman's terms what will the other players see?

**Comment by DanVanAtta** [comment]
Only one player would be listed in the player list, even if logged in multiple times. "logging in multiple times" would be less of a concept, you could have multiple windows, but you'd show and look like as just one player.

**Comment by prastle** [comment]
That's what I thought.  That is a problem! Huge one actually since every connection uses bandwidth and slows the game

**Comment by prastle** [comment]
At least if a number is shown all know this guy = twit :)

**Comment by DanVanAtta** [comment]
Latest lobby does not suffix `(1)` to player names, this is a moot problem now.
